src/js/test/index.test.js

```diff
@@ -86,3 +86,144 @@ describe('currentScreenWidth', () => {
+describe('withinBreakpoint', () => {
+    it('should return true if I pass in `>narrow` when my screen width is larger than 414', () => {
+        // Arrange
+        TestUtils.setBodyHtml(`
+        <style>
+            .c-screen-sizer {
+                content: 'narrow:414px,mid:768px,wide:1025px,huge:1280px';
+            }
+        </style>
+        `);
+        window.innerWidth = 415;
+
+        // Act
+        const breakpointMatch = withinBreakpoint('>narrow');
+
+        // Assert
+        expect(breakpointMatch).toBe(true);
+    });
+
+    it('should return true if I pass in `>=narrow` when my screen width is larger or equal to 414', () => {
+        // Arrange
+        TestUtils.setBodyHtml(`
+        <style>
+            .c-screen-sizer {
+                content: 'narrow:414px,mid:768px,wide:1025px,huge:1280px';
+            }
+        </style>
+        `);
+        window.innerWidth = 414;
+
+        // Act
+        const breakpointMatch = withinBreakpoint('>=narrow');
+
+        window.innerWidth = 415;
+        const breakpointMatchTwo = withinBreakpoint('>=narrow');
+
+        // Assert
+        expect(breakpointMatch).toBe(true);
+        expect(breakpointMatchTwo).toBe(true);
+    });
+
+    it('should return true if I pass in `<=narrow` when my screen width is smaller or equal to 414', () => {
+        // Arrange
+        TestUtils.setBodyHtml(`
+        <style>
+            .c-screen-sizer {
+                content: 'narrow:414px,mid:768px,wide:1025px,huge:1280px';
+            }
+        </style>
+        `);
+        window.innerWidth = 413;
+
+        // Act
+        const breakpointMatch = withinBreakpoint('<=narrow');
+
+        window.innerWidth = 414;
+        const breakpointMatchTwo = withinBreakpoint('<=narrow');
+
+        // Assert
+        expect(breakpointMatch).toBe(true);
+        expect(breakpointMatchTwo).toBe(true);
+    });
+
+    it('should return true if I pass in `<narrow` when my screen width is smaller than 414', () => {
+        // Arrange
+        TestUtils.setBodyHtml(`
+        <style>
+            .c-screen-sizer {
+                content: 'narrow:414px,mid:768px,wide:1025px,huge:1280px';
+            }
+        </style>
+        `);
+        window.innerWidth = 413;
+
+
+        // Act
+        const breakpointMatch = withinBreakpoint('<narrow');
+
+        // Assert
+        expect(breakpointMatch).toBe(true);
+    });
+
+    it('should return true if I pass in `=narrow` when my screen width is 414', () => {
+        // Arrange
+        TestUtils.setBodyHtml(`
+        <style>
+            .c-screen-sizer {
+                content: 'narrow:414px,mid:768px,wide:1025px,huge:1280px';
+            }
+        </style>
+        `);
+        window.innerWidth = 414;
+
+        // Act
+        const breakpointMatch = withinBreakpoint('=narrow');
+
+        // Assert
+        expect(breakpointMatch).toBe(true);
+    });
+
+    it('should return false if the passed operator is not accepted', () => {
```

Comment:
Is it worth testing for a value without a prefix as well? e.g. `withinBreakpoint('narrow');`